wallet/androidTest/de/schildbach/wallet/database/DatabaseMigrationTest.kt

```diff
@@ -54,19 +53,8 @@ open class DatabaseMigrationTest {
     @Throws(IOException::class)
     fun migrateAll() {
         // Create db and fill with data
-        testHelper.createDatabase(TEST_DB_NAME, 8).apply {
-            var values = ContentValues()
-            values.put("id", 1)
-            values.put("bestChainDate", 1633356847000)
-            values.put("bestChainHeight", BLOCKCHAIN_HEIGHT)
-            values.put("replaying", 0)
-            values.put("impediments", "")
-            values.put("chainlockHeight", 0)
-            values.put("mnlistHeight", 587091)
-            values.put("percentageSync", 100)
-            this.insert("blockchain_state", SQLiteDatabase.CONFLICT_REPLACE, values)
-
-            values = ContentValues()
+        testHelper.createDatabase(TEST_DB_NAME, 1).apply {
+            val values = ContentValues()
```

Comment:
The purpose of this test is for a sanity check.  Maybe we don't need it at all.  It was mainly to help me understand the migration process.